src/components/common/DiscountCode.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="row">
+    <div class="col-sm-offset-9 col-sm-3">
+      <ServerError :error="serverError">
+        <template slot-scope="{ graphQLError }">
+          {{ getErrorMessage(graphQLError) }}
+        </template>
+      </ServerError>
+    </div>
+    <div class="col-sm-12">
+      <form @submit.prevent="submit" class="text-right promotion-info">
+        <span class="text-uppercase promo-info-text">
+          {{ $t('promoInfo') }}
+          <img src="../../assets/img/information.png"
+               class="info-icon"
+               alt="information icon">
+        </span>
+        <input id="promo-code" type="text" name="code" v-model="code"
+               data-test="discount-code-input">
+        <LoadingButton :buttonState="buttonState"
+                       class="apply-button"
+                       data-test="apply-discount-code-button">
+          {{ $t('apply') }}
+        </LoadingButton>
+        <div v-if="cartLike.discountCodes.length > 0"
```

Comment:
`> 0` is unnecessary
https://stackoverflow.com/questions/24403732/check-if-array-is-empty-or-does-not-exist-js